btr-web/btr-common-components/app.config.ts

```diff
@@ -3,18 +3,18 @@ export default defineAppConfig({
     primary: 'bcGovBlue',
     gray: 'bcGovGray',
     formGroup: {
-      label: { base: 'block text-base font-bold text-gray-700 dark:text-gray-200' }
+      label: { base: 'block text-base font-bold py-3 text-black-700' }
```

Comment:
the text colour should pretty much always default to text-bcGovGray-700 or 900. I think for these text-bcGovGray-700 is probably in line with the other ones